backend/layers/processing/schema_migration.py

```diff
@@ -60,20 +60,22 @@ def gather_collections(self, auto_publish: bool) -> List[Dict[str, str]]:
         """
         response = []
 
-        has_revision = set()
+        has_migration_revision = set()
```

Comment:
If this is a fresh migration, nothing should have a migration revision at this point since this is the gather step, right?

Reply:
correct

Reply:
this is for cases where we run the migration more than once (due to memory errors, cancelled run, etc.)